core-services/image-mirroring/openshift/mapping_origin_4_13

```diff
@@ -275,6 +275,7 @@ registry.ci.openshift.org/origin/4.13:ptp-operator quay.io/openshift/origin-ptp-
 registry.ci.openshift.org/origin/4.13:recycler quay.io/openshift/origin-recycler:4.13 quay.io/openshift/origin-recycler:4.13.0 quay.io/openshift/origin-recycler:latest
 registry.ci.openshift.org/origin/4.13:redhat-subscription-injection-webhook quay.io/openshift/origin-redhat-subscription-injection-webhook:4.13 quay.io/openshift/origin-redhat-subscription-injection-webhook:4.13.0 quay.io/openshift/origin-redhat-subscription-injection-webhook:latest
 registry.ci.openshift.org/origin/4.13:route-controller-manager quay.io/openshift/origin-route-controller-manager:4.13 quay.io/openshift/origin-route-controller-manager:4.13.0 quay.io/openshift/origin-route-controller-manager:latest
+registry.ci.openshift.org/origin/4.13:run-once-duration-override quay.io/openshift/origin-run-once-duration-override:4.13 quay.io/openshift/origin-run-once-duration-override:4.13.0 quay.io/openshift/origin-run-once-duration-override:latest
```

Comment:
our other webhook images have webhook in the name, e.g.:
https://github.com/openshift/release/blob/facd26b628cd27d1633d6b3aea4de0b8e4d19cfd/core-services/image-mirroring/openshift/mapping_origin_4_12#L293